packages/apps/dev-wallet/src/Components/ListItem/style.css.ts

```diff
@@ -27,6 +27,7 @@ export const listItemClass = style([
   {
     border: 'none',
     flex: 1,
+    maxWidth: '100%',
```

Comment:
deosnt the flex:! take care of this already?

Reply:
somehow doesn't